### DISCUSSION TITLE
Whisper model support providing prompt?

### DISCUSSION THREAD

**Discussion by duanshiqiang** [deepgram/community#341] [General help] [answered]
I have used whisper model (not deepgram hosted) for a while and I found providing prompt can sometimes improve the transcription accuracy and also punctuation marks.

I am trying to use deepgram's whisper model API and I wonder how can I provide prompt when doing a transcibe api call?

**Comment by lukeocodes** [comment, marked as answer]
Hi @duanshiqiang we only expose features of the Whisper models that our API is set up for. Any additional features are not available

**Reply by duanshiqiang** [reply]
Any chance to add the support to the API?

**Reply by thinkmorebetter** [reply]
Seconded - is there a plan to add the `prompt` feature of the Whisper model to Deepgram's API?
